packages/react-components/react-nav-preview/src/components/Nav/Nav.types.ts

```diff
@@ -7,11 +8,36 @@ export type NavSlots = {
 /**
  * Nav Props
  */
-export type NavProps = ComponentProps<NavSlots> & {};
+export type NavProps = ComponentProps<NavSlots> & {
+  /**
+   * Tab size may change between unselected and selected states.
+   * The default scenario is a selected tab has bold text.
+   *
+   * When true, this property requests tabs be the same size whether unselected or selected.
+   * @default true
+   */
+  reserveSelectedNavGroupSpace?: boolean;
+
+  /**
+   * The value of the tab to be selected by default.
```

Comment:
I won't add corrections for all the comments.  You probably want to do a search/replace of tab and tablist :)

Reply:
yep! Thanks.